### DISCUSSION TITLE
启动速度非常慢

### DISCUSSION THREAD

**Discussion by leonlee723** [vbenjs/vue-vben-admin#984] [Q&A]
环境：Win10+webstorm.
Node: WSL2 Node 14.15
硬件：i7 8th + 16G
问题1：
![image](https://user-images.githubusercontent.com/2637309/126958415-1bdcdba8-3bbb-4679-a50c-f18f7eafa5dc.png)
ready in 35737ms，不知道是否正常
问题2：
![image](https://user-images.githubusercontent.com/2637309/126958596-03edc01b-5e4c-4261-a3f0-7215bb323941.png)
这个状态会持续大概2-3分钟才会跳转到登录页面

以上问题每次都可以复现，不是否正常？还请大家不吝赐教，非常感谢

**Comment by mynetfan** [comment]
我每天打开Webstorm第一次冷启动serve时，耗费的时间甚至比你的35s还要多，有时候能达到40秒以上。但之后每次重启serve时，基本上都在5秒左右。
页面预览也是，第一次用浏览器打开也需要挺长时间，但之后每次刷新页面也就几秒钟。

Win11+I7.10th+32G+SSD 笔记本

**Comment by baikefun** [comment]
开发时 是很长 build 后预览挺快的
